webpack.config.js

```diff
@@ -1,8 +1,58 @@
-const path = require('path');
+const Encore = require('@symfony/webpack-encore');
+const StyleLintPlugin = require('stylelint-webpack-plugin');
 
-function buildConfig(options) {
-  const env = options.prod ? 'prod' : 'dev';
-  return require(path.resolve(__dirname, `app/config/webpack/${env}.js`));
-}
+const configureCommonConfig = (enc) => {
+  if (!enc.isRuntimeEnvironmentConfigured()) {
+    enc.configureRuntimeEnvironment(process.env.NODE_ENV || 'dev');
+  }
 
-module.exports = buildConfig;
+  enc
+    .addEntry('material', './assets/material.js')
+    .addEntry('public', './assets/public.js')
+    .splitEntryChunks()
+    .enableSingleRuntimeChunk()
```

Comment:
to match the Webpack Encore Bundle recipe, see https://github.com/symfony/recipes/blob/main/symfony/webpack-encore-bundle/1.10/webpack.config.js#L28-L33